R/correctionsataglance-data.R

```diff
@@ -117,12 +117,23 @@ formatDateRange <- function(startD, endD){
 }
 
 formatDataList <- function(dataIn, type, ...){
+  
```

Comment:
This formatDataList method is a bit weird.  Its called 8 times - Are those 8 usages all so similar that they should use this same method?  It looks like it might make sense to have methods like this:
formatApprovalList()
formatMetaList()
handleCommonlistFormatting() //called by the other two methods after they have done their list-specific stuff.